safe-global/web-core
# fix: wrong message in safe creation relay

## What it solves

Resolves #1864

## How this PR fixes it
- Initiates the relay "Status" page in the PROCESSING status instead of AWAITING.

Also tackled 2 small fixes:
- Loading skeleton in dashboard's Relaying widget
- `/welcome` page card buttons

## How to test it
On create Safe through relaying, the "Please confirm the transaction with your wallet." should not appear anymore.

## Screenshots
_Skeleton_
![Screenshot 2023-04-27 at 11 21 24](https://user-images.githubusercontent.com/32431609/234904093-7366bc8e-5bc8-42ac-82a6-7336898cb496.png)

_/welcome cards_
![Screenshot 2023-04-27 at 17 03 06](https://user-images.githubusercontent.com/32431609/234904486-07d48583-f114-4a5d-bbd2-69dcb30c5142.png)


## Checklist
